XMLFormatterModel/Logging/Strategy/SimpleFileLogStrategy.cs

```diff
@@ -1,13 +1,12 @@
 ﻿using System;
 using System.IO;
-using XmlFormatter.src.Interfaces.Logging;
 
-namespace XmlFormatter.src.Logging.Strategies
+namespace XmlFormatterModel.Logging.Strategy
 {
     /// <summary>
     /// This simple logger will write the data into a given file
     /// </summary>
-    class SimpleFileLogStrategy : ILoggingStrategy
+    public class SimpleFileLogStrategy : ILoggingStrategy
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement 'IDisposable' in this class and use the 'Dispose' method to call 'Dispose' on 'streamWriter'.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)